app/src/main/res/values/strings.xml

```diff
@@ -33,9 +35,21 @@
     <string name="settings_keep_deleted_label">Keep progress of deleted files in database</string>
     <string name="settings_keep_deleted_key" translatable="false">keep_deleted</string>
     <string name="settings_keep_deleted_default" translatable="false">false</string>
-    <string name="settings_cover_from_metadata_label">Get cover image from track metadata</string>
+    <string name="settings_cover_from_metadata_label" translatable="false">Get cover image from track metadata</string>
```

Comment:
I think this setting label as well as your setting labels below should be translatable.
Only the key and the default value must be untranslatable.